psi4/share/psi4/fsapt/fsapt.py

```diff
@@ -28,13 +28,16 @@
 # @END LICENSE
 #
 
-from __future__ import print_function
+import psi4
+import numpy as np
+from typing import Dict, List, Optional, Tuple
 import sys, os, re, math, copy
+import numpy as np
 
 # => Global Data <= #
 
 # H to kcal constant
-H_to_kcal_ = 627.5095
+H_to_kcal_ = psi4.constants.hartree2kcalmol
```

Comment:
I'd be surprised if differences in results were noticeable, since we only print 3 decimal places in `fsapt.dat`. I could leave a comment in the code for those unable to get exact reproducibility.

Reply:
ah ok, maybe add a comment like `# H_to_kcal_ = 627.5095 prior to Feb 2021`